src/scripting.md

```diff
@@ -98,514 +211,551 @@ input: setCookie  %ORIGIN% foo=bar
+example: overrideHostUrl www.webpagetest.org staging.webpagetest.org
 
 <host> - host for which you want to redirect requests
 <new host> - target server to receive the redirected requests
 ```
 In this example, http://www.webpagetest.org/index.php will get rewritten to actually request http://staging.webpagetest.org/www.webpagetest.org/index.php
 
-#### addHeader
+#### `addHeader`
 Adds the specified header to every http request (in addition to the headers that exist, does not overwrite an existing header).
 Browser Support: IE, Chrome, Firefox, Safari (no SSL)
 ```markup
-usage: addHeader	<header>
-example: addHeader	Pragma: akamai-x-cache-on
+usage:   addHeader <header>
+example: addHeader Pragma: akamai-x-cache-on
 
 <header> - Full header entry to add (including label)
 ```
 
-#### setHeader
+#### `setHeader`
 Adds the specified header to every http request, overriding the header if it already exists.
 Browser Support: IE, Chrome, Firefox, Safari (no SSL)
 ```markup
-usage: setHeader	<header>
-example: setHeader	UA-CPU: none-ya
+usage:   setHeader <header>
+example: setHeader UA-CPU: none-ya
 
 <header> - Full header entry to set (including label)
 ```
 
-#### resetHeaders
+#### `resetHeaders`
 Clears any headers that were specified through addHeaders or setHeaders (in case you want to only override headers for part of a script).
 Browser Support: IE, Chrome, Firefox, Safari
 ```markup
-usage: resetHeaders
+usage:   resetHeaders
 example: resetHeaders
 ```
 
 ### Misc
 
-#### combineSteps
+#### `combineSteps`
 Causes multiple script steps to be combined into a single "step" in the results
 Browser Support: IE, Chrome, Firefox, Safari
 ```markup
-usage: combineSteps	[count]
+usage:   combineSteps [count]
 example: combineSteps
 
 [count] - Number of script steps to merge (optional, defaults to 0 which is ALL)
 Sample Script:
 
 combineSteps
-navigate	www.google.com
-navigate	www.yahoo.com
-navigate	www.aol.com
+navigate www.google.com
+navigate www.yahoo.com
+navigate www.aol.com
 ```
 
-#### if/else/endif
+#### `if`/`else`/`endif`
 Conditionally execute a block of script code based on run number or cached state for the test.  Conditional blocks can be nested.
 Browser Support: IE, Chrome, Firefox, Safari
 ```markup
-usage:  if	[cached|run]    <value>
-        else
-        endif
-
-example:    if    run    1
-            if    cached    0
-            <do something for first view of first run>
-            endif
-            else
-            <do something else for everything but first run>
-            endif
+usage:
+if [cached|run] <value>
+<commands>
+else
+<commands>
+endif
+
+example:
+if run 1
+  if cached 0
+  <do something for first view of first run>
+  endif
+else
+  <do something else for everything but first run>
+endif
 
 [cached|run] - Compare against run number or cached state
 <value> - matching run number or cached state to execute block
+<commands> - commands to run
 ```
 
-#### expireCache
+#### `expireCache`
```

Comment:
Should we mention `clearCache`? https://github.com/WPO-Foundation/webpagetest/issues/186

Reply:
Probably, yes. While it's not at an ideal state, I know a few folks have mentioned it would be useful.